config/initializers/posts_controller_decorator.rb

```diff
@@ -1,6 +1,13 @@
 Crowdblog::Admin::PostsController.class_eval do
   before_action :load_categories, only: [:edit]
 
+  def index
+    @state = params[:state]
```

Comment:
the @state variable is needed in the view in order to know what state of the blogpost is need